s3-proxy/s3-proxy/src/main/java/com/woowacourse/s3proxy/web/presentation/resolver/ExtensionValid.java

```diff
@@ -0,0 +1,12 @@
+package com.woowacourse.s3proxy.web.presentation.resolver;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.PARAMETER)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface ExtensionValid {
```

Comment:
```suggestion
public @interface ValidExtension {
```

명사+형용사 순서보다 형용사+명사 순서가 더 자연스러운 거 같습니다 👀